dsb-gui/src/main/resources/com/spectralogic/dsbrowser/gui/components/ds3panel/ds3panel.fxml

```diff
@@ -159,7 +159,7 @@
                                       AnchorPane.topAnchor="0.0">
                                     <children>
                                         <Label fx:id="createNewSessionLabel"
-                                               text="Create a New Session to Access a Black Pearl"/>
+                                               text="Create a New Session to Access a BlackPearl"/>
```

Comment:
Shouldn't this be in the resource bundle?

Reply:
fixed